gramps_webapi/api/resources/delete.py

```diff
@@ -0,0 +1,414 @@
+"""Functions for deleting objects with references."""
```

Comment:
Right! I will add the GPL header and the various copyrights from the files defining the `Delete...View` classes in Gramps, because that's where most of the code comes from.